modules/aidemBidAdapter.js

```diff
@@ -421,6 +443,11 @@ function hasValidParameters(bidRequest) {
     return false;
   }
 
+  if (!isStr(placementId)) {
+    logError('AIDEM Bid Adapter: placementId must valid string', { bidder: BIDDER_CODE, code: ERROR_CODES.PLACEMENT_ID_INVALID_VALUE });
```

Comment:
this is a breaking change and must wait for prebid 8

Reply:
linking #8539

Reply:
do you still want this?

Reply:
Yes indeed! What's the procedure for adding this?